lib/pages/new_group/new_group.dart

```diff
@@ -259,17 +273,23 @@ class NewGroupController extends State<NewGroup>
     numberSelectedImagesNotifier.addListener(() {
       if (numberSelectedImagesNotifier.value == 1) {
         Navigator.pop(context);
-        uploadAvatarNewGroupChat(
-          matrixClient: Matrix.of(context).client,
-          entity: imagePickerController.selectedAssets.first.asset
-        );
+        avatarNotifier.value = imagePickerController.selectedAssets.first.asset;
+        removeAllImageSelected();
       }
     });
   }
 
   void showImagesPickerAction({
     required BuildContext context,
   }) async {
+    final isLoading = createRoomStateNotifier.value?.fold(
+      (l) => false, 
+      (r) => r is UploadContentLoading || r is CreateNewGroupChatLoading
+    );
+    if (isLoading == true) {
+      // Disable if already uploading
```

Comment:
If you need comment sth, please use method name instead. No need to use comment.

Reply:
got it